internal/apps/dop/services/pipeline/pipeline.go

```diff
@@ -410,6 +410,12 @@ func (p *Pipeline) ConvertPipelineToV2(pv1 *apistructs.PipelineCreateRequest) (*
 		}
 	}
 	pv2.Secrets = utils.GetGittarSecrets(pv2.ClusterName, pv1.Branch, detail)
+	ownerUser, err := p.getPipelineOwnerUser(app, pv1)
+	if err != nil {
+		logrus.Errorf("get pipeline owner user error: %s", err.Error())
```

Comment:
why ignore error

Reply:
the modification may not be an erda user,

added comments